app_dart/lib/src/model/proto/internal/scheduler.proto

```diff
@@ -20,6 +20,7 @@ message SchedulerConfig {
     message Properties {
         // Generic key, value that can be passed anywhere.
         map<string, string> properties = 1;
+        map<string, string> dimensions = 2;
```

Comment:
It's a bit confusing to read dimensions under `Properties`.

Reply:
sure, renamed to PropertiesOrDimensions

Reply:
How about we call it `map<string, PlatformArgs> platform_args = 3`? Doc also needs updates.

Reply:
sounds good! umm is there a related doc for this proto?

Reply:
umm this would also involve updating every ci.yaml file to use platform_args instead of platform_properties. Do we want to proceed with changing every ci.yaml file?

Reply:
I'm against adding `Or` to the name as it's an anti-pattern (limits us to two fields). The next person to change it will need to generalize it. I think calling it `PlatformProperties` is fine, as that's what the key is already called

Reply:
I see. In that case, let's update the doc on line 16-17 to reflect the dimension `property`.

Reply:
sounds good! changed naming, updated TODO, added override and updated tests.